testing/vault/vault.initd

```diff
@@ -10,9 +10,10 @@ pidfile="/run/${SVCNAME}.pid"
 
 start_pre() {
 	checkpath -f -m 0644 -o ${SVCNAME}:${SVCNAME} "$VAULT_LOG_FILE"
+	setcap cap_ipc_lock=+ep $(readlink -f $(which vault))
```

Comment:
Yeah wasn't sure about this one, but it did fix the mlock issue. Let me update the commit.

P.S. "setcap" line came from official docs and is needed is vault is not running as root user. See here: https://www.vaultproject.io/docs/config/